packages/dcos-integration-test/extra/integration_test.py

```diff
@@ -1124,6 +1148,79 @@ def _ensure_works():
     cluster.destroy_marathon_app(app_definition['id'])
 
 
+@pytest.mark.ccm
+def test_move_external_volume_to_new_agent(cluster):
+    """Test that an external volume is successfully attached to a new agent.
+
+    If the cluster has only one agent, the volume will be detached and
+    reattached to the same agent.
+
+    """
+    hosts = cluster.slaves[0], cluster.slaves[-1]
+    test_uuid = uuid.uuid4().hex
+    test_label = 'integration-test-move-external-volume-{}'.format(test_uuid)
+    base_app = {
+        'mem': 32,
+        'cpus': 0.1,
+        'instances': 1,
+        'container': {
+            'type': 'MESOS',
```

Comment:
can we have a similar test that also exercises the DOCKER containerizer? or maybe revise this test so that it's MESOS on agent1 and DOCKER on agent2?